src/gestures/PanSession.ts

```diff
@@ -308,16 +308,20 @@ export class PanSession {
 
 function transformPoint(
     info: EventInfo,
-    transformPagePoint?: (point: Point) => Point
+    transformPagePoint?: (point: Point2D) => Point2D
 ) {
     return transformPagePoint ? { point: transformPagePoint(info.point) } : info
 }
 
+function subtractPoint(a: Point2D, b: Point2D): Point2D {
```

Comment:
I guess if it's single use it fits fine here, but I think it belongs in the geometry utils. (Don't feel strongly about this though)